mac/Focalboard/AppDelegate.swift

```diff
@@ -32,10 +32,8 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 
 	@IBAction func openNewWindow(_ sender: Any?) {
 		let mainStoryBoard = NSStoryboard(name: "Main", bundle: nil)
-		let tabViewController = mainStoryBoard.instantiateController(withIdentifier: "ViewController") as? ViewController
 		let windowController = mainStoryBoard.instantiateController(withIdentifier: "WindowController") as! NSWindowController
 		windowController.showWindow(self)
-		windowController.contentViewController = tabViewController
```

Comment:
Setting the content controller isn't necessary here because it is already configured in the storyboard and will be correctly installed when loading `WindowController`. I had to remove it because setting the new content controller caused the one that was already installed to disappear which then in turn triggered it to export the user settings via the code in `viewWillDisappear`.